examples/README.md

```diff
@@ -106,6 +106,7 @@ Example | File | Description
 `load_gltf` | [`3d/load_gltf.rs`](./3d/load_gltf.rs) | Loads and renders a gltf file as a scene
 `manual_gltf_animation_player` | [`3d/manual_gltf_animation_player.rs`](./3d/manual_gltf_animation_player.rs) | Loads and manually renders a gltf file with animations
 `many_cubes` | [`3d/many_cubes.rs`](./3d/many_cubes.rs) | Simple benchmark to test per-entity draw overhead
+`many_lights` | [`3d/many_lights.rs`](./3d/many_lights.rs) | Simple benchmark to test rendering many point lights
```

Comment:
Please put this in stress tests so I don't have to update my PR 😆

Reply:
Done.